### ISSUE TITLE
Forgot Password Link on website doesn't work

### ISSUE THREAD

**Issue by tkmorgan** [the-expanse/SideQuest#488]
On the forgot password page, when I enter my email address, a message pops up that says "Instructions Sent!", but no XHR request is submitted, no console errors appear, and no email arrives. This is a gmail address, and I'm not seeing anything in my spam folder. 

I'm using chrome version 80.0.3987.163, on Windows 10.

**Comment by sidequestlegend** [comment]
Hi there! Thanks for the report and the extra debugging detail. We currently use a websocket for api calls so you would find requests under `ws` in the network tab rather than under `xhr` - but this is all about to change. Soon ( 1-2 days ) we will be using a new http api for requests.

I have just upgraded the platform to use sendinblue as an SMTP relay which will solve this issue you are having. I'm currently using private SMTP and despite having all the correct DKIM & SPF records setup they sometimes still seem to fail. If you want to ping me on twitter or discord I can manually step you through the password reset process once i can confirm your account and get you a temporary password. 


TL;DR - Known issue, will be fixed in a day or two. Ping on twitter to get a manual reset in the mean time - https://twitter.com/SideQuestVR - https://discord.gg/Tzx7hV

Thanks for reporting!
